### ISSUE TITLE
Option to add Cosmodex versions of Alien descriptions

### ISSUE THREAD

**Issue by rezende** [fjmorel/cosmic#3]
I'd like to propose an option that would use the Alien descriptions from the Cosmodex 2.0:

https://boardgamegeek.com/thread/536444/cosmodex-20-encyclopedia-cosmic-encounter

I can do all the work myself (also including the new text files), I'd like to know if there's interest on that.

Thanks!

**Comment by fjmorel** [comment]
Definitely! I've been meaning to reach out to the author to integrate it. I'd also love to expand the app to be a reference for more than aliens.
